src/Illuminate/View/Engines/CompilerEngine.php

```diff
@@ -57,7 +57,17 @@ public function get($path, array $data = [])
         // typical PHP just like any other templates. We also keep a stack of views
         // which have been rendered for right exception messages to be generated.
         $results = $this->evaluatePath($compiled, $data);
-
+        if (config('view.template_path_hint')) {
```

Comment:
template_path_hint configuration should be added in config/view.php.
Source Code:
    'template_path_hint' => env('TEMPLATE_PATH_HINT', FALSE),